bucket/navi.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "2.16.0",
+    "description": "An interactive cheatsheet tool for the command-line",
+    "homepage": "https://github.com/casey/just",
```

```suggestion
    "homepage": "https://github.com/denisidoro/navi/",
```
Affects checkver and autoupdate